modules/Intro/Intro.Rmd

```diff
@@ -201,7 +201,7 @@ knitr::include_graphics("../../docs/images/Intro_to_R.png")
  * Lecture with slides (possibly "Interactive")
  * Lab/Practical experience
  * Two 10 min breaks each day - timing may vary
- * Jan 9-20, 2022, 1:30PM-5:00PM on Zoom
+ * Jan 9-20, 2023, 1:30PM-5:00PM on Zoom
```

Comment:
Let's try to incorporate the config here

Reply:
I added this to issue #256